sample/username_password_sample_with_classification.py

```diff
@@ -0,0 +1,103 @@
+    "scope": ["User.Read"],
+    "password": "This is a sample only. You better NOT persist your password."
+}
+
+You can then run this sample with a JSON configuration file:
+
+    python sample.py parameters.json
+"""
+
+import sys  # For simplicity, we'll read config file from 1st CLI param sys.argv[1]
+import json
+import logging
+
+import requests
+
+import msal
+import os, atexit
+from msal.exceptions import Error
+
+cache = msal.SerializableTokenCache()
+if os.path.exists("my_cache.bin"):
+    cache.deserialize(open("my_cache.bin", "r").read())
+atexit.register(lambda:
+    open("my_cache.bin", "w").write(cache.serialize())
+    # Hint: The following optional line persists only when state changed
+    if cache.has_state_changed else None
+    )
+
+
+# Optional logging
+# logging.basicConfig(level=logging.DEBUG)
+
+config = json.load(open(sys.argv[1]))
+
+# Create a preferably long-lived app instance which maintains a token cache.
+app = msal.PublicClientApplication(
+    config["client_id"], authority=config["authority"], validate_authority=False, token_cache=cache
+    # token_cache=...  # Default cache is in memory only.
+                       # You can learn how to use SerializableTokenCache from
+                       # https://msal-python.rtfd.io/en/latest/#msal.SerializableTokenCache
+    )
+
+# The pattern to acquire a token looks like this.
+result = None
+
+# Firstly, check the cache to see if this end user has signed in before
+accounts = app.get_accounts(username=config["username"])
+print("Accounts"+ str(accounts))
+if accounts:
+    logging.info("Account(s) exists in cache, probably with token too. Let's try.")
+    result = app.acquire_token_silent(config["scope"], account=accounts[0], error_response=True)
+    if result:
+        if isinstance(result, Error):
+            option = result.classification
+            if option == "basic_action":
+                print("""Condition can be resolved by user interaction 
+                during the interactive authentication flow.""")
+            elif option == "additional_action":
+                print("""Condition can be resolved by additional remedial interaction
+                with the system, outside of the interactive authentication flow.""")
+            elif option == "message_only":
+                print("""Condition cannot be resolved at this time.
+                Launching interactive authentication flow will show a message 
+                explaining the condition.""")
+                sys.exit("Abort without bothering to try acquire_token_by_xyz()")
+            elif option == "consent_required":
+                print("""Condition can be resolved by giving consent 
+                during the interactive authentication flow.""")
+            elif option == "user_password_expired":
+                print("User password expired")
+            else:
+                print("Invoke default error handling routine")
```

Comment:
Although many of this sample is derived from [the other sample of our upcoming A-B test](https://github.com/AzureAD/microsoft-authentication-library-for-python/compare/return-error-in-param?expand=1#diff-350433198f3b1800ace53a8ba389538c), here this sample does not trigger a "default error handling routine" (which would be to call `acquire_token_by_xyz()`). This would not be a meaningful A-B test. Might give it another shot?